pkg/cache/cache_test.go

```diff
@@ -508,7 +509,7 @@ func TestCacheClusterQueueOperations(t *testing.T) {
 					ResourceGroups:                []ResourceGroup{},
 					NamespaceSelector:             labels.Everything(),
 					FlavorFungibility:             defaultFlavorFungibility,
-					Usage:                         FlavorResourceQuantities{},
+					Usage:                         resources.FlavorResourceQuantitiesFlat{}.Unflatten(),
```

Comment:
Seems unnecessary, given that we are going to rename FlavorResourceQuantitiesFlat to FlavorResourceQuantities in the next PR.

Reply:
`find ./pkg/ -type f | xargs sed -i "s/resources.FlavorResourceQuantitiesFlat{}.Unflatten()/resources.FlavorResourceQuantities{}/g"`